modules/core/test/v2/unit/wallets.ts

```diff
@@ -94,6 +98,23 @@ describe('V2 Wallets:', function() {
       .reply(200, {});
       yield wallets.add({ label: 'label', enterprise: 'enterprise', type: 'custodial' });
     }));
+
+    it('creates a wallet with custom gasPrice', co(function *createWalletCustomGasPrice() {
+      const ethBitGo = new TestBitGo({ env: 'mock' });
+      ethBitGo.initializeTestVars();
+      const ethWallets = ethBitGo.coin('teth').wallets();
+      nock(bgUrl)
+        .post('/api/v2/teth/wallet', function(body) {
+          body.type.should.equal('custodial');
+          body.gasPrice.should.equal(20000000000);
```

Comment:
question: are you sure gasPrice isn't coming back as a string? most of our stuff is BigNumber at the API level